src/graph_compute/numa_init.h

```diff
@@ -0,0 +1,53 @@
+#ifndef NUMA_INIT_H_
+#define NUMA_INIT_H_
+
+#include <sched.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <sys/mman.h>
+#include <string>
+#include <cstdio>
+#include <cassert>
+#include <cstring>
+#include <algorithm>
+// #include "./io.h"
```

Comment:
Can you update this pull request with the change? You need to commit and push to its branch, `pr_rollback`.